pkg/model/publisher.go

```diff
@@ -11,7 +11,6 @@ const (
 	publisherUnknown Publisher = iota // must be first
 	PublisherNoop
 	PublisherIpfs
-	PublisherFilecoin
```

Comment:
Removing this means that people who have in the past submitted Estuary or S3 jobs now have data that is interpreted incorrectly, e.g. previously Estuary was 4 but now S3 has that value, so places where the numbers are used (existing data) will be wrong. Can we be sure that noone is using these numbers? If not, can we just use `PublisherEstuary = 4` and make sure that 3 isn't used?

Reply:
In a very similar vein to https://github.com/bacalhau-project/bacalhau/pull/2552#discussion_r1224572846: These will go away when https://github.com/bacalhau-project/bacalhau/issues/2543 lands. 

The pattern of using enums/constants for these values comes at the cost of them being hard to change later, the work to complete #2543 will replace these with CID's backed by a schema similar to how https://github.com/bacalhau-project/bacalhau/issues/2542 will replace engine enums with CIDs.

Reply:
This shouldn't be a problem as we always marshal and unmarshal publisher types to their string value before sending them around